atom/browser/native_window_mac.mm

```diff
@@ -918,6 +918,7 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
   }
 
   if (transparent()) {
+    NSLog(@"Setting transparent");
```

Comment:
I believe this is not related to the PR.

Reply:
Woops, had that in my staged env 🏃